### ISSUE TITLE
clean up git after leanproject get-mathlib-cache

### ISSUE THREAD

**Issue by kckennylau** [leanprover-community/mathlib-tools#54]
Feature request: have `leanproject get-mathlib-cache` execute at the end:
```shell
git reset --hard [commit id specified by leanpkg.toml]
git clean -fd
```

**Comment by gebner** [comment]
I'm very **strongly** ***against*** this change.  Most of my mathlib clones have untracked files in them that should not be committed (like bug reports from Zulip, etc.).  This change would delete them without warning.

**Comment by PatrickMassot** [comment]
We certainly wan't run `git clean -fd` when editing mathlib itself.

**Comment by PatrickMassot** [comment]
I think Kenny wanted to run this inside `_target/deps/mathlib`, which is currently completely removed when updating.

**Comment by kckennylau** [comment]
yeah this is meant to be for a repo that depends on mathlib, rather than mathlib itself

**Comment by PatrickMassot** [comment]
Fixed in b1e6d66
